src/block/icon-list/schema.js

```diff
@@ -86,6 +96,25 @@ export const iconListAttributes = {
 		type: 'number',
 		default: '',
 	},
+
+	// List item borders.
+	listItemBorderStyle: {
+		type: 'string',
+		default: '',
+	},
+	listItemBorderWidth: {
+		stkResponsive: true,
+		type: 'number',
+		default: '',
+	},
+	listItemBorderColor: {
+		type: 'string',
+		default: '#00000066',
```

Comment:
The default should be `''` here. Please set the border color CSS to `#00000066` instead of setting the default value here. Adding the default here will make the block generate extra css styles